enterprise/admin/views.py

```diff
@@ -134,15 +134,14 @@ def _build_context(self, request, customer_uuid):
         search_keyword = self.get_search_keyword(request)
         linked_learners = self.get_enterprise_customer_user_queryset(search_keyword, customer_uuid)
         pending_linked_learners = self.get_pending_users_queryset(search_keyword, customer_uuid)
+        enrollment_url = '/api/enrollment/v1/'
 
         context = {
             self.ContextParameters.ENTERPRISE_CUSTOMER: enterprise_customer,
             self.ContextParameters.PENDING_LEARNERS: pending_linked_learners,
             self.ContextParameters.LEARNERS: linked_learners,
             self.ContextParameters.SEARCH_KEYWORD: search_keyword or '',
-            self.ContextParameters.ENROLLMENT_URL: getattr(settings,
-                                                           'ENTERPRISE_PUBLIC_ENROLLMENT_API_URL',
-                                                           settings.ENTERPRISE_ENROLLMENT_API_URL),
```

Comment:
Can we remove these settings entirely now if we aren't using them?

Reply:
I'll check if they are used elsewhere and if not will delete them

Reply:
added new setting for LMS_ENROLLMENT_API_PATH but leaving other settings for now. We can clean them up in another PR